style.css

```diff
@@ -0,0 +1,675 @@
+}
+
+h3::first-letter {
+	text-transform: uppercase;
+}
+
+.item-row p::first-letter {
+	text-transform: uppercase;
+}
+
+.item-row {
+	padding: 24px;
+	background-color: rgba(255, 255, 255, 1);
+	border: 1px solid rgba(220, 233, 226, 1);
+	transition: .5s;
+}
+
+.item-row:hover {
+	transform: scale(1.5);
+	cursor: pointer; 
+}
+
+.doctor-image {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	flex-wrap: wrap;
+}
+
+.medicine-text {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	padding-left: 67px;
+	flex-wrap: wrap;
+}
+
+.medicine-text p::first-letter {
+	text-transform: uppercase;
+}
+
+.contato-info {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	flex-wrap: wrap;
+}
+
+.contato-info h2 {
+	padding-bottom: 40px;
+}
+
+.pacient-image {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	flex-wrap: wrap;
+
+}
+
+.contacts li {
+	display: flex;
+	flex-wrap: wrap;
+}
+.contacts {
+	padding-bottom: 40px;
+}
+
+.contacts li div {
+	padding-right: 10px;
+}
+
+.button p {
+	padding-left: 18px;
+}
+
+address {
+	font-style: normal;
+	text-transform: capitalize;
+	color: rgba(73, 80, 87, 1);
+	font-weight: 400;
+	font-size: 16px;
+	line-height: 150%;
+}
+
+.contacts li a {
+	color: rgba(73, 80, 87, 1);
+	font-weight: 400;
+	font-size: 16px;
+	line-height: 150%;
+}
+
+.footer {
+	background-color: rgba(0, 133, 111, 1);
+}
+
+.footer-container {
+	max-width: 1120px;
+	/* height: 268px; */
+	padding-top: 80px;
+	padding-bottom: 80px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	flex-wrap: wrap;
+	column-gap: 672px;
+	
+}
+
+.social-network-item {
+	display: flex;
+	flex-wrap: wrap;
+}
+
+.logo-footer {
+	padding-bottom: 24px;
+}
+
+.company-info .rights-company-item {
+	padding-bottom: 15px;
+	font-weight: 400;
+	font-size: 14px;
+	line-height: 18px;
+	color: rgba(255, 250, 241, 1);
+}
+
+.company-info .rights-item::first-letter {
+	text-transform: uppercase;
+}
+
+.company-info .rights-item {
+	font-weight: 400;
+	font-size: 14px;
+	line-height: 18px;
+	color: rgba(255, 250, 241, 1);
+}
+
+.youtube, .facebook {
+	padding-left: 40px;
+}
+
+[class*="-container"] {
+	margin: 0px 24px;
+}
+
+
+/* =========================================================== */
+
+@media (max-width: 375px) {
+	.header-menu, .link-item {
+		display: none;
+	}
+	.header-container {
+		padding: 27px 24px 27px 24px;
+}
+	.assistência-médica-container {
+		display: flex;
+		align-items: center;
+		flex-wrap: wrap;
+		position: relative;
+}
+	.médica-text-item, .médica-text {
+		text-align: center;
+		flex-wrap: wrap;
+		position: relative;
+}
+	h1 {
+		font-size: 34px;
+}
+	button, .woman-image {
+		margin: 0 auto;
+}
+	.items {
+		border: 1px solid rgba(220, 233, 226, 1);
+		bottom: 30px;
+		display: grid;
+		grid-template-rows: 1fr 1fr 1fr;
+		grid-template-columns: 1fr;
+		padding: 40px 80px 40px 80px;
```

Comment:
I would put simple `padding: 40px` as a width will change during resize and eventually this 80px paddings will cause scrollbar appearing. If you use simply 40px it will be the minimal padding (but can be more if width allows it)